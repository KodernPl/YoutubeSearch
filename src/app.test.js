import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from './app';

jest.mock('./utils/youtube-search', () => {
  return jest.fn();
});

const renderComponent = () => {
  return shallow(<App />);
};

test('renders correctly', () => {
  jest.mock('./config', () => ({
    YT_API_KEY: '1',
  }));
  const component = renderComponent();
  expect(toJson(component)).toMatchSnapshot();
});
