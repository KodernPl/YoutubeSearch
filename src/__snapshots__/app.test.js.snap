// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<app__AppWrapper>
  <SearchBar
    onSearchTermChange={[Function]}
  />
  <app__Main>
    <app__VideoBox>
      <VideoDetail
        video={null}
      />
    </app__VideoBox>
    <VideoList
      onVideoSelect={[Function]}
      videos={Array []}
    />
  </app__Main>
</app__AppWrapper>
`;
